en/docs/learn/ai/agent/agent-with-connector-tools.md

```diff
@@ -0,0 +1,233 @@
+# Integrating agents with external endpoints using prebuilt connectors
```

Comment:
Have more shorter title. And also make sure file name, title name and left nave text is same always.

Reply:
Updated